requirements/extras/azureblockblob.txt

```diff
@@ -0,0 +1,3 @@
+azure-storage>=0.36.0
```

Comment:
It is not advisable to use non-exact tags for package versions, as they might lead to inconsistencies among installations and CI results. Could you please use the most recent usable version?

Reply:
Could you provide some more guidance on what you require here? The azure-storage SDK has a bit of an odd versioning scheme where all versions below 0.36.0 are considered deprecated and the "new" SDK is versions 0.36.0 and above (see [compatibility section on the official repository](https://github.com/Azure/azure-storage-python#compatibility)). As such, this dependency statement basically means "use any of the new SDK versions". Would you like to pin this to a specific version instead?

Reply:
Exactly, my suggestion was to pin the version for the new dependencies to the exact latest:

```
azure-storage==0.36.0
azure-common==...
azure-storage-common==...
```

Reply:
Alright, done in ce12efd.